config/services.yml

```diff
@@ -45,6 +45,9 @@ services:
     DemosEurope\DemosplanAddon\Contracts\CurrentContextProviderInterface:
         class: demosplan\DemosPlanCoreBundle\Logic\CurrentContextProvider
 
+    DemosEurope\DemosplanAddon\Contracts\CurrentUserInterface:
```

Comment:
Is this really necessary?

Reply:
> Is this really necessary?

it is jup, otherwise we have the usual error that no DemosEurope\DemosplanAddon\Contracts\CurrentUserIn terface service found and that we have to alias it to the demosplan existing service

Reply:
weird this should be possible to handle globally